### DISCUSSION TITLE
Saving hash data - moving to new location

### DISCUSSION THREAD

**Discussion by tbounce85** [airdcpp/airdcpp-windows#121] [Q&A]
Hey, within a few months I am moving to another house. I have a massive share of over 60TB. Is there a way to save my hash data and load it onto my new location somehow? It contains internal + external hard disks + NAS drives ... Or will I have to rehash all my data?

**Comment by maksis** [comment]
You might find this useful: https://github.com/airdcpp-web/airdcpp-webclient/issues/437
